### ISSUE TITLE
Calling `.add` multiple times on relations is broken

### ISSUE THREAD

**Issue by mstniy** [parse-community/Parse-SDK-JS#2056]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse JS SDK](https://github.com/parse-community/Parse-SDK-JS/releases). 

### Issue Description
Calling `.add` multiple times on a relation adds only the object mentioned in the first call.

### Steps to reproduce
```
it('can do consecutive adds', done => {
    const ChildObject = Parse.Object.extend('ChildObject');
    const childObjects = [];
    for (let i = 0; i < 2; i++) {
      childObjects.push(new ChildObject({ x: i }));
    }
    const parent = new Parse.Object('ParentObject');
    parent.save().then(parentAgain => {
      Parse.Object.saveAll(childObjects)
        .then(() => {
          for (const child of childObjects) {
            parentAgain.relation('child').add(child);
          }
          return parentAgain.save();
        })
        .then(() => {
          return parentAgain.relation('child').query().find();
        })
        .then(results => {
          assert.equal(results.length, 2);
          const expectedIds = new Set(childObjects.map(r => r.id));
          const foundIds = new Set(results.map(r => r.id));
          assert.deepEqual(expectedIds, foundIds);
          done();
        });
    });
  });
```

### Actual Outcome
Only the first child object is added to the relation

### Expected Outcome
Both of the child objects should be added to the relation

###  Environment

Server
- Parse Server version: [alpha](https://github.com/parse-community/parse-server/commit/d3087ed69f9745848729951171657b79d3192469)
- Operating system: Ubuntu
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): local

Database
- System (MongoDB or Postgres): MongoDB
- Database version: `6.0.11`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): local

Client
- Parse JS SDK version: [alpha](https://github.com/parse-community/Parse-SDK-JS/commit/fea8edfa6c342419d06f60c686591baec2f51fc4)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.3.1-alpha.2](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.3.1-alpha.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/Parse-SDK-JS/releases/tag/5.0.0)
